src/datachain/lib/file.py

```diff
@@ -375,6 +378,7 @@ def get_file_type(
             size=size,
             vtype=vtype,
             location=location,
+            last_modified=last_modified,
```

Comment:
`is_latest` needs to be passed in as well. (it's never False in tests because we're not testing cloud versioning enough)